README.md

```diff
@@ -640,6 +661,125 @@ or
 ecs-cli compose --ecs-params my-ecs-params.yml service up --launch-type FARGATE
 ```
 
+#### Using Route53 Service Discovery
+
+With the ECS CLI, you can create a Service that uses [Route53 auto naming for service discovery](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-discovery.html). Service Discovery requires a Service Discovery Service, and a DNS Namespace. When you enable Service Discovery with the ECS CLI, a new Service Discovery Service is always created using CloudFormation. For the DNS Namespace, you have the option of using an existing public or private DNS Namespace, or letting the ECS CLI create a private DNS Namespace for you using CloudFormation. Creation of a Public DNS Namespaces is not supported with the ECS CLI. Please note that only a single DNS Namespace may be used with Service Discovery.
+
+##### Enabling Service Discovery
+
+###### Specifying Values
+
+There are many input parameters for Service Discovery, however, for most of them, the ECS CLI will choose reasonable defaults. Here are the default values and explanations listed with the ECS Params input schema:
+
+```
+version: 1
+run_params:
+  service_discovery:
+    // container name and port are the container that will be used for Service Discovery
+    container_name: string            // No Default, required if using SRV records
+    container_port: string            // No Default, required if using SRV records
+    private_dns_namespace:
+      vpc: string                     // Required if ID is not specified
+      id: string                      // Allows you to specify an existing namespace by ID
+      name: string                    // DNS name for the namespace, if it already exists, the CLI will use, if not, the CLI creates one
```

Comment:
Possible rephrase: DNS name for private namespace. Used to specify an existing namespace; if one does not exist with this name, the ecs-cli will create it.